### DISCUSSION TITLE
Overflow bin

### DISCUSSION THREAD

**Discussion by Tomoya-Iizawa** [scikit-hep/cabinetry#449] [Q&A] [answered]
Hello,

Is there any way to include overflow bin in the fit with the cabinetry framework?

Best,
Tomoya

**Comment by alexander-held** [comment, marked as answer]
Hi, currently the answer to that is no. See also #374. This should not be too difficult to add, there are two questions for the implementation:
- how to configure this (maybe something like a "use_flow" setting per region, with values like "underflow", "overflow", "both" or so),
- how to handle this in plots (to indicate that flow bins enter the plot).

It probably would be easiest to add an option to merge under-/overflow into the outermost bins and ignore the plotting complication for now. If you have any feedback about the interface or functionality, that could be useful!

**Reply by Tomoya-Iizawa** [reply]
Hi,
Thank you for your answer, and sorry for the late reply.
We implemented to add an additional bin as overflow bin.
